### PR TITLE
Mcs 713 Cora

### PR DESCRIPTION
**Pull request by cedorman**
Implementation of the CORA scaling using Ray.

### CODE REVIEW

**Review by kdrumm — COMMENTED** [review]
LGTM, although now that some other PRs were merged, we should merge dev in.  Github seems to not be factoring that into the diff.

File: autoscaler/ray_cora_aws.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+cluster_name: cora-clark-v4
+max_workers: 8                       # 2 is the default
+
+# Note that ray only seems to use the first availability zone, so if there are issues with that 
+# one, just need to put a different one first.
+provider:
+  type: aws
+  region: us-east-1
+  availability_zone: us-east-1c, us-east-1b, us-east-1d, us-east-1a
```
Comment (kdrumm) [inline autoscaler/ray_cora_aws.yaml:15]: This doesn't work for me unless we remove the spaces.

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]
Didn't run it, but the code LGTM. Good use of comments.

File: autoscaler/ray_cora_aws.yaml (hunk 1 above)
Comment (cedorman) [reply autoscaler/ray_cora_aws.yaml:15]: Thanks.  Noticed that was fixed in your alignment.

File: autoscaler/ray_cora_aws.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+#
+#   Configuration file for:
+#         Team:   Baseline
```
Comment (rartiss55) [inline autoscaler/ray_cora_aws.yaml:3]: Comment update -- should say "CORA" here.

**Pull request merged**